### PR TITLE
PXP-616: [CLI] Log Entries is not reset after a new namespace or a new version is selected

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary
There are few bugs found during my discoveries, so I made a PR to fix them together.

Ticket: [PXP-616](https://mindvalley.atlassian.net/browse/PXP-616)

### Fixed
- [x] Fix the `log entries` and `log_entries_length` is not reset after new namespace or new version is selected
- [x] Fix the duplicated gcloud API call after a new namespace is selected
- [x] Fix the gcloud API not being called after a new namespace or a new version is selected when the tailing is disabled

[PXP-616]: https://mindvalley.atlassian.net/browse/PXP-616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/152#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mfauzaan — APPROVED** [review]
LGTM!

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/152#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/152#pullrequestreview-1652189627" title="Approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
